core/db/db.go

```diff
@@ -52,59 +60,25 @@ type OpenchainDB struct {
 	StateDeltaCF *gorocksdb.ColumnFamilyHandle
 	IndexesCF    *gorocksdb.ColumnFamilyHandle
 	PersistCF    *gorocksdb.ColumnFamilyHandle
+	dbState      dbState
+	mux          sync.Mutex
 }
 
 var openchainDB *OpenchainDB
-var isOpen bool
+var once sync.Once
 
-// CreateDB creates a rocks db database
-func CreateDB() error {
```

Comment:
I saw it. A question for line 81 to line 86. It seems open a rocksdb on the dbpath, then close it.I cannot catch it